suite-native/atoms/src/RoundedIcon.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactNode } from 'react';
+
+import { Color } from '@trezor/theme';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import {
+    CryptoIcon,
+    EthereumTokenIcon,
+    FlagIcon,
+    Icon,
+    AnyIconName,
+    isCryptoIconType,
+    isEthereumTokenIconType,
+    isFlagIconType,
+    isIconType,
+} from '@trezor/icons';
+
+import { Box } from './Box';
+
+type RoundedIconProps = {
+    name: AnyIconName;
```

Comment:
`AnyCoinName` does not accept token symbol names for which we don't have an icon. Is it intentional?